packages/drivers/odsp-driver/src/createFile/createFile.ts

```diff
@@ -26,7 +26,7 @@ import { createOdspUrl } from "./../createOdspUrl.js";
 import { EpochTracker } from "./../epochTracker.js";
 import { getHeadersWithAuth } from "./../getUrlAndHeadersWithAuth.js";
 import { OdspDriverUrlResolver } from "./../odspDriverUrlResolver.js";
-import { getApiRoot } from "./../odspUrlHelper.js";
+import { checkForInternalFarmType, getApiRoot } from "./../odspUrlHelper.js";
```

Comment:
SHould we rename this to checkForServiceFarmType or checkForServerFarmType? Internal does not make sense to me.